docs/source/sensors/index.rst

```diff
@@ -4,17 +4,40 @@ Cameras that can be used with USGSCSM
 
 .. toctree::
   Cassini
-  Dawn
-  Galileo
-  Hayabusa
-  Juno
   LunarReconnaissanceOrbiter
   MarsExpress
   MarsReconnaissanceOrbiter
   Messenger
+  Selene
```

Comment:
Oddly, even JAXA uses the nickname first: https://www.kaguya.jaxa.jp/index_e.htm

So, Kaguya (SELENE). 

I have no idea why they do this. I would consider this form though to match JAXA / NASA (e.g, [NSSDCA](https://nssdc.gsfc.nasa.gov/nmc/spacecraft/display.action?id=2007-039A))